readthedocs/projects/constants.py

```diff
@@ -355,3 +355,6 @@
 )
 
 GITHUB_PR_PULL_PATTERN = 'pull/{id}/head:external-{id}'
+
+# Git provider names
+GITHUB = 'GitHub'
```

Comment:
This isn't a good name, since it has specialized caps. It should likely be `GITHUB_BRAND` or something,

Reply:
`GITHUB_PROVIDER_NAME` maybe?